R/Filter.R

```diff
@@ -116,8 +116,6 @@ Filter = R6Class("Filter",
 
       if (task$nrow == 0L) {
         self$scores = shuffle(set_names(rep.int(NA_real_, length(fn)), fn))
-      } else if (task$ncol == 0L) {
-        self$scores = set_names(numeric(), character())
```

Comment:
@mllg 	`task$ncol == 0` is never TRUE because the target column can never be removed. I removed the condition therefore. Is that ok?

Reply:
`length(task$feature_names) == 0`?